src/pages/ConfigurationPage/ConfigurationPage.jsx

```diff
@@ -1,7 +1,14 @@
-import React from "react";
+import { useContext } from "react";
+import { DarkModeContext } from "./DarkModeContext";
 
 const ConfigurationPage = () => {
-  return <h2>configuration</h2>;
+  const { darkMode, toggleTheme } = useContext(DarkModeContext);
+
+  return (
+    <button className="lightSwitch" onClick={toggleTheme}>
+      {darkMode ? "light Mode" : "Dark Mode"}
```

Comment:
Pay attention to the details.
```suggestion
      {darkMode ? "Light Mode" : "Dark Mode"}
```